tasks/task166_ClariQ_sentence_generation.json

```diff
@@ -0,0 +1,10887 @@
+{
+    "Contributors": "Mirali Purohit",
+    "Source": [
+        "ClariQ (Dataset - https://github.com/aliannejadi/ClariQ) (Paper - https://arxiv.org/pdf/2009.11352.pdf)"
+    ],
+    "Categories": "Sentence Generation",
+    "Definition": "This task mainly performs the task of clarifying queries/questions for dialogue systems. A user is asking an ambiguous question/query (where ambiguous question is a question to which one can return > 1 possible answers). Particularly, you are given a question/query in natural language and based on that your task is to ask a good clarifying question to understand the query more precisely for the given question/query.",
```

Comment:
- "This task mainly performs the task of"? Simplify. 
 - "ambiguous question is a question": Simplify. 
 - "one can return > 1 possible answers": "return" might be unclear to those who don't know computers. Rephrase.

Reply:
Corrected the definition, please review again.